libs/common/src/lib/policies/site.policy.ts

```diff
@@ -23,6 +24,7 @@ export class SitePolicy extends UserPermissionsPolicy {
         ];
         if (projectIds.length > 0) {
           this.builder.can("read", Site, { projectId: { $in: projectIds } });
+          this.builder.can("delete", Site, { projectId: { $in: projectIds } });
```

Comment:
The same restriction applies for non-admins deleting sites, so let's do the override in the site processor as well.

Reply:
changed accordingly